utils/bootstrap-node

```diff
@@ -67,6 +67,7 @@ while true; do
         --mkfs)              do_mkfs=mkfs; shift ;;
         --mkfs-only)         do_mkfs=mkfs-only; shift ;;
         -p|--phase)          phase=$2; shift 2 ;;
+        --xprt)              xprt=$2; shift 2 ;;
```

Comment:
If we don't pass  the transport with hctl bootstrap, can we make libfab as default.

Reply:
I strongly recommend to rename `--xprt` to something human-readable.
What is it, BTW? My first impression was that it stands for the word "expert" :-)

Please address this in a new PR if possible.

Reply:
@madhavemuri , made libfab as default tansport.
@knekrasov, xprt is a well known abbreviation used for transport, check `struct m0_net_xprt` at https://github.com/Seagate/cortx-motr/blob/main/net/net.h.

We can surely make it more explicit, may be `--transport` or `--transport-type`, but even that might mean something else without the help message for the same I believe.